lighthouse-core/audits/screenshot-thumbnails.js

```diff
@@ -0,0 +1,119 @@
+/**
+ * @license
+ * Copyright 2017 Google Inc. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+'use strict';
+
+const Audit = require('./audit');
+const jpeg = require('jpeg-js');
+
+const NUMBER_OF_THUMBNAILS = 10;
+const THUMBNAIL_HEIGHT = 100;
```

Comment:
it'd be better to constrain on width, so we can be sure that 10thumbnails has a reliable total width